### ISSUE TITLE
[FEATURE] Failing if a nuget package with same version already exists

### ISSUE THREAD

**Issue by AnmolSinha1201** [brandedoutcast/publish-nuget#43]
This usually happens when we already have a nuget package with the same version already hosted on nuget. AFAIK, since we are not allowed to "replace" a package on nuget, is it possible for you to fail instead? That way, if this is already part of a pipeline, the pipeline would actually fail and it would be a reminder for us to update the version.

**Comment by brandedoutcast** [comment]
Strange 🤔 with the change detection in place it shouldn't event try to push a package with same version could you provide the action log to get a better idea?

**Comment by AnmolSinha1201** [comment]
Yeah, it doesn't push the package with same version. What I am asking for is, to throw an exception too, if the same version exists. That way, the pipeline would fail. You can hide this behind a flag so it remains compatible with earlier versions.

**Comment by Marfusios** [comment]
+1 would be useful to fail whole build if we forget to increment version

**Comment by hhoangnl** [comment]
Would absolutely love this feature!

**Comment by gregsdennis** [comment]
Secondarily to @Marfusios's comment, it'd be great if we could get a version that runs on PRs (no publish) to _check_ whether the version already exists.
